### ISSUE TITLE
Non-radius checkpoints

### ISSUE THREAD

**Issue by iznogod** [opencj-dev/server-gsc#112]

**Comment by Yocairo** [comment]
What would this imply? A checkpoint without a radius would seem infinitesimal to me.

**Comment by iznogod** [comment]
Yeah, it's triggered by something in the map script instead (trigger_use_touch activation etc)

**Comment by fnkcj** [comment]
A lot of maps have teleports near the ending that take the player to some sort of end room or back to spawn. Those would work well as the final checkpoint in those maps.

**Comment by Yocairo** [comment]
I don't necessarily agree with that @fnkcj , the checkpoints should be placed appropriately (i.e. before the teleporter) if it teleports you to spawn. No need to add a weird non-radius checkpoint at spawn that only triggers if you went past the previous checkpoint

**Comment by iznogod** [comment]
The checkpoint would not be at spawn, but at the teleport, and trigger upon using the teleport (ie trig waittill("trigger", player) event)

**Comment by Yocairo** [comment]
If you're talking about integrating them into map scripts (which is where teleporters reside, because they can have any name), then I don't like it. Don't want to update map scripts every time an update comes out that may or may not add teleporters or change their names
